.travis.yml

```diff
@@ -54,6 +54,7 @@ script:
   - moodle-plugin-ci savepoints
   - moodle-plugin-ci mustache
   - moodle-plugin-ci grunt
+  - moodle-plugin-ci phpdoc
```

Comment:
Please add this same line to the `.travis.dist.yaml` file.

Reply:
Addressed in a6a0de7